go.mod

```diff
@@ -33,6 +33,8 @@ require (
 	github.com/onsi/gomega v1.31.1
 	github.com/otiai10/copy v1.14.0
 	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.8.4
+	github.com/tj/assert v0.0.3
```

Comment:
Please remove these deps and use gomega, this the test framework we use for Flux.

Reply:
thank you for pointing that out @stefanprodan